Lib/fontParts/base/font.py

```diff
@@ -158,7 +158,7 @@ def _get_path(self, **kwargs):
 
     # save
 
-    def save(self, path=None, showProgress=False, formatVersion=None):
+    def save(self, path=None, showProgress=False, formatVersion=None, fileStructure=None):
```

Comment:
In `defcon.Font.save()` and `fontTools.ufoLib.UFOWriter()` the argument is called `structure`. Maybe it should be called that here as well?

Reply:
briefly discussed with @benkiel: `fileStructure` implies its about the file itself and not about any other kind of structure. 

ufoLib also calls it `fileStructure` [see](https://github.com/fonttools/fonttools/blob/master/Lib/fontTools/ufoLib/__init__.py#L293)

similar to formatVersion is not version

and Im out of arguments :)

Reply:
I see the argument for making it the same as ufoLib/defcon, but if I had to go back, I'd change the name there to `fileStructure`. I _think_ that may be more obvious to folks who don't know as much about defcon/ufoLib than just `structure`, but we're also splitting hairs so if it's more important to align with defcon, that's cool

Reply:
I'm ok with `fileStructure`.

Reply:
+1 for `fileStructure`. I don't know why I used `structure` in defcon. Maybe it should change there.